appeals/api/src/server/endpoints/lpa-questionnaires/__tests__/lpa-questionnaires.test.js

```diff
@@ -231,6 +232,54 @@ describe('lpa questionnaires routes', () => {
 				expect(
 					databaseConnector.lPAQuestionnaireIncompleteReasonOnLPAQuestionnaire.update
 				).not.toHaveBeenCalled();
+
+				// eslint-disable-next-line no-undef
+				expect(mockSendEmail).toHaveBeenCalledTimes(1);
```

Comment:
do we need a `expect(mockSendEmail).toHaveBeenCalledWith` here too?

Reply:
thought I'd make the test a tad more robust, but no, strictly speaking it's not needed